### ISSUE TITLE
Pets skills cant be deactivated 

### ISSUE THREAD

**Issue by MKorin** [WarEmu/WarBugs#8968]
Tested on my SH but its prolly on all pets, theres no way to turn off single pets skill.

**Comment by Luth-RoR** [comment]
Yes, pet abilities are still not working completely as they did on the mythic server. I think with client access fixes to the mythic mess are more likely.

**Comment by ozarmidrashim** [comment]
Visual only.

**Comment by Neutraz** [comment]
Still buggy, very annoying on pet classes like WL. It is possible to deactivate the skill, but due to a visual bug it doesn't show that it is deactivated.

A fix would be awesome. Would be even more awesome if the server would remember the status after the pet is killed / resummoned (wasn't even on live i think). Thanks.

Same bug report: https://github.com/WarEmu/WarBugs/issues/6718
